aries-site/src/examples/templates/FormChildObject/ChildHeader.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Header, Heading, Text } from 'grommet';
+import { FormUp, FormDown } from 'grommet-icons';
+
+export const ChildHeader = ({
+  // annotationIds are for documentation demo purposes only.
+  // Remove from product implementation.
+  annotationIds,
+  collectionName,
+  index,
+  level,
+  name,
+  open,
+  summary,
+  ...rest
+}) => {
+  const [background, setBackground] = useState(null);
+  const borderStyle = { side: 'top', color: 'border-weak' };
+
+  return (
+    <Header
+      id={annotationIds?.container}
+      background={background}
+      border={borderStyle}
+      height={{ min: 'xxsmall' }}
+      onMouseEnter={() => setBackground('background-contrast')}
+      onMouseLeave={() => setBackground(null)}
+      pad="small"
+      {...rest}
+    >
+      <Box>
```

Comment:
do we need this `box` maybe use `<>`

Reply:
I believe we need this Box bc otherwise the Heading and summary would render in a row.

Reply:
your right my bad